add keypoints to 05 (#402)
datacarpentry/python-ecology-lesson: _episodes/05-merging-data.md

    - "Employ `to_csv` to export a DataFrame in CSV format."
    - "Join DataFrames using common fields (join keys)."
keypoints:

>     - "`Merge` and `concat` can be used to recombine subsets from a DataFrame, or even data from different files."

Reviewer: Python is case-sensitive, so... maybe update to:
> "Panda's `.merge` and `.concat` methods are used to (re-)combine subsets of a DataFrame or even data from different files."

Author: ```suggestion
    - "`merge` and `concat` can be used to (re)-combine subsets from a DataFrame, or even data from different files."
```

I left them in without the dot notation, but lowercase. There are other points where we refer to methods without dot notation.

Reviewer: Yeah, I'm fine with that. I think it _might_ be useful to use the same approach across multiple lessons but this is not important at this point.

Reviewer: shall it, perhaps, be `combine subsets of a DataFrame` ?

Author: ```suggestion
    - "`Merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
```

>     - "Pandas' `merge` and `concat` can be used to (re)-combine subsets from a DataFrame, or even data from different files."

Author: ```suggestion
    - "Pandas' `merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
```

>     - "DataFrames can be combined using join keys with the `join` function."

Reviewer: "join keys" seems unclear / jargonny

Author: ```suggestion
    - "DataFrames can be combined using based on column names with the `join` function."
```

Author: like so?

Reviewer: Maybe... alternative:
> `join` function combines DataFrames based on index or a column.

Author: I like that!

>     - "Joining two DataFrames can be done in multiple ways (left, right, and inner) according to what data must be in the final DataFrame."

Reviewer: according to -> depending on

Author: ```suggestion
    - "Joining two DataFrames can be done in multiple ways (left, right, and inner) depending on what data must be in the final DataFrame."
```

Reviewer: Did you decide to not include this one?

>     - "`to_csv` can be used to write out DataFrames in CSV format."

Reviewer: I suggest spelling out CSV: Comma-Separated Values format

Author: We define CSV in 02, so I think it's OK to leave.

Author: Forgot to click 'commit'
